apps/root/src/pages/earn/components/strategies-table/components/columns/index.tsx

```diff
@@ -183,8 +183,10 @@ export const strategyColumnConfigs: StrategyColumnConfig<StrategiesTableVariants
   {
     key: StrategyColumnKeys.SAFETY,
     label: <FormattedMessage description="earn.all-strategies-table.column.safety" defaultMessage="Safety" />,
-    renderCell: (data) => getStrategySafetyIcon(data.riskLevel),
-    getOrderValue: (data) => Object.keys(StrategyRiskLevel).length - data.riskLevel,
+    renderCell: (data) => (data.riskLevel ? getStrategySafetyIcon(data.riskLevel) : '-'),
+    getOrderValue: (data) =>
+      Object.keys(StrategyRiskLevel).length -
+      Object.values(StrategyRiskLevel).findIndex((rskLvl: StrategyRiskLevel) => rskLvl === data.riskLevel),
```

Comment:
I think we should check if data.riskLevel exists, otherwise findIndex will return -1 and will ordered incorrectly

Reply:
Actually the getComparator function was already handling non defined values, so just made a change to return undefined if the riskLevel is not there :)